src/api/api.js

```diff
@@ -25,24 +26,77 @@ async function createEvent(event) {
 
   let res;
   try {
-    res = await protectedResourceAxios.post(`${process.env.VUE_APP_API_DOMAIN}/api/v1/protected/events/`, body);
+    res = await protectedResourceAxios.post('/api/v1/protected/events/', body);
```

Comment:
Did you mean to get rid of the domain?

Reply:
Yes, the axios instance has the domain configured into it already